### PR TITLE
Add various bits of front matter

### PR DESCRIPTION
**Pull request by jasongrout**
This copies over various bits of frontmatter from @blink1073's frontmatter example repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,87 @@
-# jupyter-js-input-area
+jupyter-js-input-area
+=======================
 
-Javascript APIs for Jupyter input areas
+[![Build Status](https://travis-ci.org/jupyter/jupyter-js-input-area.svg)](https://travis-ci.org/jupyter/jupyter-js-input-area?branch=master)
+[![Coverage Status](https://coveralls.io/repos/jupyter/jupyter-js-input-area/badge.svg?branch=master&service=github)](https://coveralls.io/github/jupyter/jupyter-js-input-area?branch=master)
 
-## Development
+Front matter for a Jupyter jupyter-js-based application.
```
Comment (blink1073) [inline README.md]: Needs an update

Comment (jasongrout) [inline README.md]: Thanks.  Fixed.

File: test/system.conf.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    'phosphor-signaling': 'node_modules/phosphor-signaling/lib/index.js',
+    'phosphor-widget': 'node_modules/phosphor-widget/lib/index.js',
+    'phosphor-messaging': 'node_modules/phosphor-messaging/lib/index.js',
+    'phosphor-queue': 'node_modules/phosphor-queue/lib/index.js',
+    'phosphor-arrays': 'node_modules/phosphor-widget/node_modules/phosphor-arrays/lib/index.js',
+    'phosphor-observablelist': 'node_modules/phosphor-widget/node_modules/phosphor-observablelist/lib/index.js',
+    'phosphor-nodewrapper': 'node_modules/phosphor-widget/node_modules/phosphor-nodewrapper/lib/index.js',
+    'codemirror': 'node_modules/codemirror/lib/codemirror.js',
+    'codemirror/*': 'node_modules/codemirror/*',
+    'diff-match-patch': 'node_modules/diff-match-patch/index.js',
```
Comment (jasongrout) [inline test/system.conf.js:18]: This is terrible.  There has to be a better way to give the paths for every package and its dependencies.

Comment (jasongrout) [inline test/system.conf.js:18]: For example, possibly the system-npm plugin that steal uses could help here.  Problem is that somehow we have to kick off steal loading the initial package.json file.

**Comment by jasongrout** [comment]
@blink1073 - I think this is ready to merged as an improvement, and then we iterate again.

**Comment by blink1073** [comment]
Iterating.

**Pull request merged**